book-worm-2010/book-worm-fe
# Feature/oauth

### What does this PR do? 
>Clearly and concisely describe the feature. 

Connect to Google OAuth for user authentication on login page.  Login and Logout components were added to the src folder to add functionality and styling.

![Screen Shot 2021-04-17 at 7 46 29 PM (2)](https://user-images.githubusercontent.com/67291333/115131546-25aea980-9fb6-11eb-9dc1-75ac5c8e7d27.png)

### Is this a feature or Fix/Refactor?  

Feature 

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

Some issues were encountered with initial setup and connecting with the token.  The problem was solved when adding the localhost address to the token on the backend.

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?

util, Login, and Logout folders in src

### How should this be tested?
>What steps need to be taken to confirm functionality? 

Go to project directory and run `npm start` in terminal to open the app in the browser.  You should be able to sign in with your google account. Open the console to see the success response.  

### Steps To Take
>Any questions and possible steps moving forward.

Render the home page upon sign in.  Add animation to worm svg on login.
